vite.config.ts

```diff
@@ -14,6 +14,9 @@ export default defineConfig({
       '@': join(__dirname, 'src')
     }
   },
+  optimizeDeps: {
+    exclude: ['@storybook/react-vite']
+  },
```

Comment:
was breaking storybook for me and deleting node_modules didn't fix. only this did